Amazon.Ion.ObjectMapper.Test/IonObjectSerializerTest.cs

```diff
@@ -24,13 +25,14 @@ public void SerializesAndDeserializesObjectsWithIncludeFields()
         public void SerializesObjectsWithIgnoreNulls()
         {
             var serializer = new IonSerializer(new IonSerializationOptions {IgnoreNulls = true});
-            var motorcycle = new Motorcycle {canOffroad = true};
+            var motorcycle = new Motorcycle(null, null, true, DateTime.Now);
```

Comment:
Why this change? I believe it's because some properties no longer have setters?

Reply:
Constructors are required to set the readonly property `PurchaseDate`.

Reply:
Upon further investigation, the `{ field = value }` style is allowed if there is a parameterless constructor. It was only a problem when I first introduced the parameterized constructor before adding the parameterless constructor.
I have now reverted the style back to what it was before.